Sources/Connection.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Connection.swift
+//  C7
+//
+//  Created by Scott Byrns on 4/1/16.
+//
+//
+
```

Comment:
we used to have a MIT header but @tannernelson didn't like them haha so he proposed we removed then all cause a single `LICENSE` file in the repo is enough. everybody agreed so we now have no headers at all in the source files.